framework/elastic_query_builder.rb

```diff
@@ -143,7 +143,12 @@ def construct_es_query_term filter_key, value
       {
         multi_match: { query: value, type: flag, fields: multi_match_fields }.compact
       }
-    when "term", "fuzzy", "prefix", "wildcard", "regexp"
+    when "fuzzy"
+      multi_match_fields = fields == ["_all"] ? nil : fields
```

Comment:
I would recommend setting to '*.*' instead of nil if we're matching all, making that explicit instead of relying on the elastic default for no fields specified. That may avoid some confusion if `index.query.default_field` is set to something different

Reply:
Based myself on the case above it where it is also set to nil, but will change it to `*.*`